### ISSUE TITLE
Datomic Cloud & Muuntaja deps

### ISSUE THREAD

**Issue by eneroth** [metosin/muuntaja#121]
FYI, 

In order for Muuntaja to work with Datomic Cloud, these deps (as of right now) need to be included in the root:

```Clojure
{com.fasterxml.jackson.core/jackson-core                {:mvn/version "2.10.1"}
 com.fasterxml.jackson.core/jackson-databind            {:mvn/version "2.10.1"}
 com.fasterxml.jackson.datatype/jackson-datatype-jsr310 {:mvn/version "2.10.1"}}
```

And the same need to be excluded from Muuntaja:

```
{metosin/muuntaja {:mvn/version "0.6.8"
                   :exclusions  [com.fasterxml.jackson.core/jackson-core
                                 com.fasterxml.jackson.core/jackson-databind
                                 com.fasterxml.jackson.datatype/jackson-datatype-jsr310]}}
```

Love Muuntaja though, it's great.

**Comment by onetom** [comment]
I've just posted a related comment on an existing thread on the Datomic forum, which asks for a `jackson-core` upgrade:
https://forum.datomic.com/t/jackson-core-needs-an-update/1862

**Comment by onetom** [comment]
Would it make sense to provide the very core of `muuntaja`, without any specific default encodings, so we can plug our own choice of JSON library in?

**Comment by eneroth** [comment]
In order to not break existing users of the library, would the core then be broken out into a new library on which Muuntaja depends?

**Comment by onetom** [comment]
> In order to not break existing users of the library, would the core then be broken out into a new library on which Muuntaja depends?

yes, that's how I would imagine it. I would expect that library to provide `muuntaja.core`, but unfortunately that NS already exists and depends on `muuntaja.format.{json,transit}`.

`muuntaja.format.edn` might make sense to keep as a default, since it only depends on `clojure.edn`, which is bundled with Clojure itself, but it's probably more flexible to provide a completely clean slate, for the sake of simplicity.
